### PR TITLE
cache: Remove one 0 from hash name check to cache more URLs

### PR DESCRIPTION
**Pull request by octonawish-akcodes**

### CODE REVIEW

File: kcidb/cache/__init__.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def store(self, url):
         object_name = self._format_object_name(url)
 
         # Cache every 4096th URL only for the trial period
```
Comment (spbnick) [inline kcidb/cache/__init__.py:53]: Looks good, but please update the comment as well.

Comment (spbnick) [reply kcidb/cache/__init__.py:53]: Yep!

**Review by spbnick — CHANGES_REQUESTED** [review]

File: kcidb/cache/__init__.py (hunk 1 above)
Comment (octonawish-akcodes) [reply kcidb/cache/__init__.py]: done

**Comment by octonawish-akcodes** [comment]
done

**Review by spbnick — APPROVED** [review]
LGTM! Thank you, Abhishek. Let's see how CI behaves. Those changed hash requirements could have affected tests.

**Pull request merged**